charts/airbyte/values.yaml

```diff
@@ -0,0 +1,380 @@
+  ## choice for the user. This also increases chances charts run on environments with little
+  ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+  ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  ## @param scheduler.resources.limits [object] The resources limits for the scheduler container
+  ## @param scheduler.resources.requests [object] The requested resources for the scheduler container
+  resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 200m
+    ##    memory: 1Gi
+    limits: {}
+    ## Examples:
+    ## requests:
+    ##    memory: 256Mi
+    ##    cpu: 250m
+    requests: {}
+
+  ## @param scheduler.nodeSelector [object] Node labels for pod assignment
+  ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+
+  ## @param scheduler.tolerations [array] Tolerations for scheduler pod assignment.
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: []
+
+
+## @section Pod Sweeper parameters
+
+podSweeper:
+  ## @param podSweeper.image.repository The image repository to use for the pod sweeper
+  ## @param podSweeper.image.pullPolicy The pull policy for the pod sweeper image
+  ## @param podSweeper.image.tag The pod sweeper image tag to use
+  image:
+    repository: bitnami/kubectl
+    pullPolicy: IfNotPresent
+    tag: latest
+
+  ## @param podSweeper.podAnnotations [object] Add extra annotations to the podSweeper pod
+  ##
+  podAnnotations: {}
+
+  ## Pod Sweeper app resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## We usually recommend not to specify default resources and to leave this as a conscious
+  ## choice for the user. This also increases chances charts run on environments with little
+  ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+  ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  ## @param podSweeper.resources.limits [object] The resources limits for the podSweeper container
+  ## @param podSweeper.resources.requests [object] The requested resources for the podSweeper container
+  resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 200m
+    ##    memory: 1Gi
+    limits: {}
+    ## Examples:
+    ## requests:
+    ##    memory: 256Mi
+    ##    cpu: 250m
+    requests: {}
+
+  ## @param podSweeper.nodeSelector [object] Node labels for pod assignment
+  ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+
+  ## @param podSweeper.tolerations [array] Tolerations for podSweeper pod assignment.
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: []
+
+
+## @section Server parameters
+
+server:
+  ## @param server.replicaCount Number of server replicas
+  replicaCount: 1
```

Comment:
Same for the scheduler comment above. We're planning on making this horizontally scalable but haven't yet.